ci/ansible/roles/pulp-crane/tasks/main.yml

```diff
@@ -22,17 +35,42 @@
     state: true
     persistent: true
 
-- name: Install Crane configuration file
-  copy:
-    src: crane.conf
-    dest: /etc/crane.conf
-    owner: root
-    group: root
-    mode: 0644
+- block:
+  - name: Install Crane configuration file
+    copy:
+      src: crane.conf
+      dest: /etc/crane.conf
+      owner: root
+      group: root
+      mode: 0644
+
+  - name: Open firewall port for Crane (firewalld)
+    firewalld:
+      port: "5000/tcp"
+      state: enabled
+      permanent: true
+      immediate: true
+  when:
+    - not (ansible_distribution == "RedHat" and ansible_distribution_major_version|int == 6)
```

Comment:
Hmm, you bringing this up makes me realize that perhaps the other one should be the same as this, because 
"not (this AND that)" == "(not this OR not that)" and I'm not sure the other line actually will do the right thing on Fedora. I'll make it consistent and it and test it out on Fedora 25.